### DISCUSSION TITLE
Question about SSL?

### DISCUSSION THREAD

**Discussion by icantpronouncethis** [debauchee/barrier#2004] [Q&A]
I'm fairly new with the tech vocab.
Currently I have barrier set up with ssl disabled.
I am planning to keep barrier running at all times.
I was wondering if that is something I should be worried about.

**Comment by TommieWaters** [comment]
on your own network at home you be fine. just dont use it on a public wifi or one you dont own. an attacker could send mesages to you devices and control them but like they would need to know the communication structure which someone could easy enough figure out with some ecpertise. i myself cant get it working without ssl
